app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row">
+  <div class="col-10 offset-1">
+    <h2 align="center">
+      <strong><%= @organization.name %></strong> repositories list:
+    </h2>
+    <br>
+    <table class="table table-hover">
+      <thead>
+      <tr>
+        <th scope="col">Repo Name</th>
+        <th scope="col">License Name</th>
+        <th scope="col">License Spdx Id</th>
+      </tr>
+      </thead>
+      <tbody>
+      <%- @organization.repositories.each do |repository| %>
+        <%- if repository.license_name.nil? %>
```

Comment:
Please use `spdx_id` is a primary selector here, and only use `license_name` for display purposes